src/comparisons.jl

```diff
@@ -1,7 +1,42 @@
-# This file is inspired from Polyhedra.jl
+# Some functions in this file are inspired from Polyhedra.jl
 
-const ABSZTOL(N::Type{<:AbstractFloat}) = N(10) * sqrt(eps(N))
-const ABSZTOL(N::Type{Rational{INNER}}) where {INNER} = zero(N)
+# struct to contain the tolerances for a given numberic type
+mutable struct TOL{N<:Number}
+    rtol::N
+    ztol::N
+    atol::N
+end
+
+# global Float64 tolerances
+const F64 = Float64
+const tF64 = Type{Float64}
```

Comment:
I would not introduce these constants for just a few lines. They make reading the code harder.

Reply:
I disagree, since these constants are repeated several times, i added them to make the code look simpler. But i won't insist, it would be genuine bikeshedding 😆